frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -300,12 +271,25 @@ export function PureSidebar({
   );
 }
 
-export function useSidebarItem(itemName: string | null) {
+export function useSidebarItem(
```

Comment:
Please add documentation to functions that are changed, even if they didn't have them before.